docs/using-dapr-ambient/README.md

```diff
@@ -0,0 +1,184 @@
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.redis
+  version: v1
+  metadata:
+  - name: keyPrefix
+    value: name
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    secretKeyRef:
+      name: redis
+      key: redis-password
+auth:
+  secretStore: kubernetes
+EOF
+```
+
+Create the PubSub component applying this resource to Kubernetes by running:
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: notifications-pubsub
+spec:
+  type: pubsub.redis
+  version: v1
+  metadata:
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    secretKeyRef:
+      name: redis
+      key: redis-password
+auth:
+  secretStore: kubernetes
+EOF
+```
+
+Once we have the PubSub component configured, we can register Subscritions to define who and where notifications will be sent when new messages arrive to a certain topic.
+
+Create the Subscription component applying this resource to Kubernetes by running:
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: dapr.io/v1alpha1
+kind: Subscription
+metadata:
+  name: notifications-subscription
+spec:
+  topic: notifications 
+  route: /notifications
+  pubsubname: notifications-pubsub
+EOF
+```
+
+## Installing Dapr Ambient and all applications
+
+Finally, let's install Dapr Ambient and three applications that uses the Dapr StateStore and PubSub components.
+
+Install Dapr Ambient running this:
+
+```sh
+  helm package  chart/dapr-ambient
+  helm install my-ambient-dapr-ambient dapr-ambient-1.9.5.tgz --set ambient.appId=read-values --set ambient.proxy.remoteURL=read-values-svc:8080
+```
+
+Let's deploy the apps:
+
+This are normal/regular Kubernetes applications, using Deployments and Services.
+```sh
+  kubectl apply -f https://raw.githubusercontent.com/salaboy/dapr-example-apps/main/apps.yaml
+```
+
+If you want to see the implementation's detail, you [can access this repository](https://github.com/salaboy/dapr-ambient-examples).
+
+### Saving using the write-values application
+
+Let's create a value on the store:
+
+```sh
+  kubectl port-forward svc/write-values-svc 8080:8080
+```
+
+Send a request to the application:
+
+```sh
+  curl --request POST \
+  --url 'http://localhost:8080/?value=10'
+``` 
+
+You can see the log using `kubectl logs -f <pod>`
+
+At this point the `subscriber` application has been received the notification from `dapr-ambient`. You can see this, with the same way, using `kubectl logs -f <pod>`.
+
+### Getting the average fom read-values application
+
+The `read-values` applications gets all values from StateStore and calculates the average.
+
+```sh
+  kubectl port-forward svc/read-values-svc 8888:8080
+```
+
+After, you can make a request to `read-values-svc`:
+
+```sh
+  curl http://localhost:8888
+```
+
+The response should looks like it:
+
+```
+10
+```
+
```

Comment:
@mcruzdev I think we are missing the subscription.. we need to add that to the demo.. it is quite important.. you need to add the subscriber app to the examples too.